components/com_kunena/admin/controllers/templates.php

```diff
@@ -252,18 +252,156 @@ function uninstall()
 		$this->setRedirect(KunenaRoute::_($this->baseurl, false));
 	}
 
+	function chooseless()
+	{
+		$jinput = JFactory::getApplication()->input;
+		$template = $jinput->get('id', '', 'method', 'cmd');
```

Comment:
You can just do here $this->app->input->get('id', '', 'method', 'cmd'); you don't need to call 
$jinput = JFactory::getApplication()->input;